resolver/GlobalPass.cc

```diff
@@ -137,6 +138,11 @@ void resolveTypeMembers(core::GlobalState &gs, core::SymbolRef sym,
 
     if (sym.data(gs)->isClassOrModuleClass()) {
         for (core::SymbolRef tp : sym.data(gs)->typeMembers()) {
+            // AttachedClass is covariant, but not controlled by the user.
+            if (tp.data(gs)->name == core::Names::Constants::AttachedClass()) {
+                continue;
+            }
+
```

Comment:
Have you tried to come up with an example that abuses this to cause a variance violation that should probably have a static error message but which doesn't?

Reply:
Or is it Just Not Possible and if so could you sketch out why?

Reply:
I haven't yet, though my gut feeling is that the defaulting behavior of `externalType` should rule out funny business there.